### ISSUE TITLE
Vulnerable dependency & purpose

### ISSUE THREAD

**Issue by manglano** [hoccy/hoccy#3]
This software might still be a good idea, like people form critical masses all the time (gaming conventions, protests, Critical Mass, uh, even like Starbucks in the morning)--I think Foursquare's Swarm has really filled the niche for people open to meeting in technology-rich environments though.

I'm much better equipped to contribute to frontend & backend than I once was.

I dunno--thought I'd register some thoughts here rather than tinkering with it or wiping out the repository or what have you. Hope everyone's doing well, sorry I'm an insular and lazy dude sometimes, it's my nature. Hope you're both doing well

-Dave

**Comment by nelsonpecora** [comment]
Haha no worries. Yeah, I think the original idea for this is good, but I didn't really know how to handle the potential harassment vectors. Now that other people (Swarm, BumbleBFFs, etc) are trying to solve it, I'd let them handle it.

**Comment by manglano** [comment]
Yeah, sounds good. I was thinking about that the other day...Moderation is
a bigger obstacle to most social app use than the app development itself.
That problem used to be somewhat uncommon (or the internet's default
response was like "oh, a shock image, ha ha, weird and so gross") but now
it's such an issue that YouTube is in the news for a lack of content
moderation.

Which is also kinda like, "How would YouTube moderate?" because they have
literally millions of hours of content.

Oh well...One less thing to work on!
